udata_front/theme/gouvfr/assets/js/components/AdminTable/AdminDataservicesTable/AdminDataservicesTable.vue

```diff
@@ -0,0 +1,108 @@
+<template>
+  <AdminTable :loading>
+    <thead>
+      <AdminTableTh
+        @sort="(direction: SortDirection) => $emit('sort', 'title', direction)"
+        :sorted="sorted('title')"
+        scope="col"
+      >
+        {{ t("Dataservice title") }}
+      </AdminTableTh>
+      <AdminTableTh>
+        {{ t("Status") }}
+      </AdminTableTh>
+      <AdminTableTh>
+        {{ t("Access") }}
+      </AdminTableTh>
+      <AdminTableTh>
+        {{ t("Created at") }}
+      </AdminTableTh>
+      <AdminTableTh>
+        {{ t("Rate Limiting") }}
+      </AdminTableTh>
+      <AdminTableTh>
+        {{ t("Availability") }}
+      </AdminTableTh>
+    </thead>
+    <tbody>
+      <tr v-for="dataservice in dataservices">
+        <td>
+          <AdminContentWithTooltip>
+            <a class="fr-link fr-reset-link" :href="getDataserviceLinkToAdmin(dataservice)">
+              <TextClamp :text="dataservice.title" :auto-resize="true" :max-lines="2"/>
+            </a>
+          </AdminContentWithTooltip>
+        </td>
+        <td><AdminBadge :type="getStatus(dataservice).type">{{ getStatus(dataservice).label }}</AdminBadge></td>
+        <td><AdminBadge :type="getAccess(dataservice).type">{{ getAccess(dataservice).label }}</AdminBadge></td>
+        <td>{{ formatDate(dataservice.created_at) }}</td>
+        <td>{{ dataservice.rate_limiting }}</td>
+        <td>{{ dataservice.availability }}%</td>
```

Comment:
We could put a condition on `dataservice.availability` to not show a column with just a "%" if there is no availability filled.